libsoftwarecontainer/component-test/filegateway_componenttest.cpp

```diff
@@ -69,3 +69,43 @@ TEST_F(FileGatewayTest, TestActivateWithMinimalValidConf) {
     ASSERT_TRUE(gw->setConfig(jsonConfig));
     ASSERT_TRUE(gw->activate());
 }
+
+TEST_F(FileGatewayTest, MalformedPath) {
+// Mount a path which is malformed, e.g. too many "/"
+    const std::string config =
+        "[{"
+        "  \"path-host\": \"//tmp//filename.txt\""
+        ", \"path-container\": \"///filename.txt\""
+        "}]";
+    loadConfig(config);
+
+    // Is this ok? Will the extra "/" will be removed?
```

Comment:
Yes this is more or less on the level of a TODO. I saw the printout during the test and couldn't see if this path (with extra slashes) was what was being using for the bindmount command. I wanted to check up if we remove extra slashes (couldn't find it in our code, so perhaps it is the help functions we use). So it was a reminder to myself to check up on it.